pkg/plugin/aggregation/aggregator_test.go

```diff
@@ -286,56 +289,58 @@ func TestAggregation_RetryWindow(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
-		agg := NewAggregator(dir, expected)
-		// Shorten retry window for testing.
-		agg.retryWindow = testRetryWindow
-		testDataPath := "./testdata/fakeLogData.txt"
-		testDataReader, err := os.Open(testDataPath)
-		if err != nil {
-			t.Fatalf("Could not open test data file: %v", err)
-		}
-		defer testDataReader.Close()
-
-		var r io.Reader
-		if tc.simulateErr {
-			r = iotest.TimeoutReader(testDataReader)
-		} else {
-			r = strings.NewReader("foo")
-		}
+		t.Run(tc.desc, func(t *testing.T) {
```

Comment:
This was a table-driven test but was probably implemented before subtests in golang.

I've just indented everything; I originally had an error here and couldn't tell why because it was running all test cases in one go.